tools/generator/tests/test_AnDOGenerator.py

```diff
@@ -52,6 +51,7 @@ def setUp(self):
         self.sesNumber = '100'
         self.customSesField = 'test'
         self.sesID = f'{self.date}_{self.sesNumber}_{self.customSesField}'
+        self.ephys = 'ephys'
```

Comment:
I don't think you are using this anywhere. Better remove it then.

Reply:
Done =)